### ISSUE TITLE
skSK: No celebration of `Shrove Monday/Tuesday` and `{{day}} before Epiphany`

### ISSUE THREAD

**Issue by tukusejssirs** [pejulian/romcal#14]
We never use `Shrove Monday/Tuesday` in the Liturgy. Even in folk (-lore) traditions of Slovakia, I could find `Shrove Tuesday` equivalent (sk: `Fašiangový utorok`), but still it is not used throughout Slovakia. We used `Monday of the 6th week of Ordinary time` on 12 Feb 2018 instead. I need to be sure if I can use `{{day}}` and `{{week}}` variables in these keys, too—this would solve this problem.

Likewise, we never use `{{day}} before Epiphany`. Instead, we use `{{day}} in Christmastide` instead.

**Comment by tukusejssirs** [comment]
In order to clarify this question, I’d like to say that I would like to remove `after Epiphany` altogether from `skSk.mjs` if possible. Yhat would require to insure that `{{day}} in Christmastide` would be used up until the Epiphany.

**Comment by pejulian** [comment]
Hi @tukusejssirs 

I have added a new feature called the `drop` property that allows for the exclusion of non relevant celebrations from national calendar output. I have added the drop property for Shrove Monday and Shrove Tuesday in `src/calendars/slovakia.mjs` so that they are excluded when calling `calendarFor` for Slovakia. The corresponding tests and usage instructions in `README.md` have been added too.
